examples/error_handling.py

```diff
@@ -0,0 +1,80 @@
+import discord
+from discord.ext import commands
+import traceback
+
+description = '''
+A basic bot featuring error handling for command errors
+'''
+
+# Constructing the bot with a prefix and the description
+bot = commands.Bot(command_prefix='!', description=description)
+
+# Example command to showcase a potential error raised.
+# The important ones to see are `MissingRequiredArgument` and `BadArgument`.
+# `MissingRequiredArgument` will be raised when a required argument is not passed.
+# Bad Argument will be raised in this case when the arg cannot be converted to an `int`.
+@bot.command()
+async def add(ctx, arg1: int, arg2: int):
+    added = arg1 + arg2
+    await ctx.send('{} + {} = {}'.format(arg1, arg2, added))
```

Comment:
```suggestion
async def divide(ctx, dividend: float, divisor: float):
    divided = dividend / divisor
    await ctx.send('{} / {} = {}'.format(dividend, divisor, divided))
```
This is perhaps a good way of showcasing the unwrapping of `CommandInvokeError` - to catch `ZeroDivisonError`.